tests/common_testing/alfasim_sdk_common_testing/filled_case_descriptions.py

```diff
@@ -507,6 +507,7 @@
     esp_reference_density=Scalar(1000.0, "kg/m3"),
     user_defined_esp_table=TABLE_PUMP_DESCRIPTION,
     esp_parameters=constants.EspParameters.Catalog,
+    enable_density_correction=False,
```

Comment:
You probably want to chance the `esp_thermal_efficiency_model`. Even disabled it will be dumped, and it is important that to check this.

Reply:
Can you be more especific about that?

Reply:
Set the value of `esp_thermal_efficiency_model` when creating this case description to a non default value.

Reply:
does this comes from master?
```
class PumpThermalEfficiencyModel(Enum):
    """
    Defines the ESP thermal efficiency model
```

@Gualcar 
Is this eps only related? Shouldn't ne prefix of with `eps_` to make that clear?

Reply:
I think @rafacerq will know it properly. But, this `PumpThermalEfficiencyModel` is an attribute of the class `Pump` (the parent class for all pumps), so I think it should be maintained as general term.

Reply:
If that is the case the enum doc string must be updated.

Reply:
Yeah, it could be replaced by `Defines the pump thermal efficiency model`.